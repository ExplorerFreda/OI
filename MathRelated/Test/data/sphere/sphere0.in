2
6751.000000 19873.000000
984.500000 13278.500000
6020.500000 4359.000000
