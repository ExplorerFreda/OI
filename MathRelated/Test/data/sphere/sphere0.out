9114.820 11987.498
